### ISSUE TITLE
Bug report: Inaccurate information about StaFi staking pool

### ISSUE THREAD

**Issue by Tore19** [ethereum/ethereum-org-website#9740]
### Describe the bug

Hi, I'm from StaFi. We believe there is some information about StaFi from this page(https://ethereum.org/en/staking/pools) that needs to be updated.

<img width="355" alt="image" src="https://user-images.githubusercontent.com/10955745/225953985-82bda41e-a550-4fa2-8f8c-9ec4d015994f.png">

Any validator can run nodes through StaFi, so it should be permissionless.

Please check the rETH APP: https://app.stafi.io/validator/reth/choose-validator

<img width="1233" alt="image" src="https://user-images.githubusercontent.com/10955745/225954416-8cd7a212-30c9-4770-b73a-414b2995194b.png">

### To reproduce

1. Go to 'https://ethereum.org/en/staking/pools'
2. Scroll down to 'Explore staking pools'
3. See error

### Expected behavior

It should be permissionless about StaFi staking pool.

**Comment by vishal590** [comment]
hi @corwintines, assign this issue to me. I can work on this issue.

**Comment by wackerow** [comment]
@Tore19 Thanks for posting this... I'll be dedicating time over the next two weeks to get through reviewing PRs related to staking products and will include this.

@vishal590 Appreciate the offer here. I'm going to self assign this though for review of the request being made... if you'd like to look into any of this yourself in the meantime and post your findings in support of or refuting anything mentioned, happy to take a look at anything you find. 🙏

**Comment by vishal590** [comment]
@wackerow ok

**Comment by Tore19** [comment]
@wackerow  Great! Thanks!

**Comment by wackerow** [comment]
@Tore19 Just took a look at this, and overall sounds good! Great to see more options for permissionless node operation.

One question / some feedback...
<img width="592" alt="image" src="https://user-images.githubusercontent.com/54227730/226988618-518a98b8-295b-4cf5-836b-b53c131d02a1.png">
This confused me at first when I saw "SOLO", which I implied I would need to stake alone, vs only contributing 12 ETH and filling in the rest with pooled stakers. 

That doesn't necessarily prevent this PR from moving forward, but wanted to offer that bit of feedback. I'll toss up a PR to update this flag.
